scripts/exregional_novarcldanl.sh

```diff
@@ -0,0 +1,315 @@
+
+bkpath=${CYCLE_DIR}/INPUT
+if [ -w ${bkpath}/gfs_data.tile7.halo0.nc ]; then  # Use background from INPUT
+  ln_vrfy -s ${bkpath}/sfc_data.tile7.halo0.nc      fv3_sfcdata
+  ln_vrfy -s ${bkpath}/gfs_data.tile7.halo0.nc      fv3_dynvars
+  ln_vrfy -s ${bkpath}/gfs_data.tile7.halo0.nc      fv3_tracer
+else                                               # Use background from RESTART
+  ln_vrfy -s ${bkpath}/fv_core.res.tile1.nc         fv3_dynvars
+  ln_vrfy -s ${bkpath}/fv_tracer.res.tile1.nc       fv3_tracer
+  ln_vrfy -s ${bkpath}/sfc_data.nc                  fv3_sfcdata
+fi
+
+#
+#-----------------------------------------------------------------------
+#
+# link/copy observation files to working directory
+#
+#-----------------------------------------------------------------------
+
+PROCESS_BUFR_PATH=${CYCLE_DIR}/process_bufr
+
+obs_file=${PROCESS_BUFR_PATH}/LightningInFV3LAM.dat
+if [ -r "${obs_file}" ]; then
+   cp_vrfy "${obs_file}" "LightningInFV3LAM.dat"
+else
+   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+fi
+
+obs_file=${PROCESS_BUFR_PATH}/NASALaRC_cloud4fv3.bin
+if [ -r "${obs_file}" ]; then
+   cp_vrfy "${obs_file}" "NASALaRC_cloud4fv3.bin"
+else
+   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+fi
+
+obs_file=${PROCESS_BUFR_PATH}/fv3_metarcloud.bin
+if [ -r "${obs_file}" ]; then
+   cp_vrfy "${obs_file}" "fv3_metarcloud.bin"
+else
+   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+fi
+
+#-----------------------------------------------------------------------
+#
+# Build namelist 
+#
+#-----------------------------------------------------------------------
+
+cat << EOF > gsiparm.anl
+
+ &SETUP
+  iyear=${YYYY},
+  imonth=${MM},
+  iday=${DD},
+  ihour=${HH},
+  iminute=00,
+ /
+ &RAPIDREFRESH_CLDSURF
+   dfi_radar_latent_heat_time_period=20.0,
+   metar_impact_radius=10.0,
+   metar_impact_radius_lowCloud=4.0,
+   l_pw_hgt_adjust=.true.,
+   l_limit_pw_innov=.true.,
+   max_innov_pct=0.1,
+   l_cleanSnow_WarmTs=.true.,
+   r_cleanSnow_WarmTs_threshold=5.0,
+   l_conserve_thetaV=.true.,
+   i_conserve_thetaV_iternum=3,
+   l_cld_bld=.true.,
+   l_numconc=.true.,
+   cld_bld_hgt=1200.0,
+   build_cloud_frac_p=0.50,
+   clear_cloud_frac_p=0.10,
+   iclean_hydro_withRef_allcol=1,
+   i_gsdcldanal_type=6,
+   i_gsdsfc_uselist=1,
+   i_lightpcp=1,
+   i_gsdqc=2,
+   l_saturate_bkCloud=.true.,
+ /
+EOF
+
+
+
+#
+#-----------------------------------------------------------------------
+#
+# Copy the executable to the run directory.
+#
+#-----------------------------------------------------------------------
+#
+EXEC="${EXECDIR}/fv3sar_novarcldana.exe"
+
+if [ -f $EXEC ]; then
+  print_info_msg "$VERBOSE" "
+Copying the noVar Cloud Analysis executable to the run directory..."
+  cp_vrfy ${EXEC} ${workdir}/fv3sar_novarcldana.exe
+else
+  print_err_msg_exit "\
+The executable specified in EXEC does not exist:
+  EXEC = \"$EXEC\"
+Build executable and rerun."
+fi
```

Comment:
This comment is relevant in many of the ex-scripts, and sometimes multiple times.

Since the name of the executable is left unchanged, please consider the following alternative to address the duplication of information (name of file is here multiple times):

```
cp_vrfy ${EXEC} ${WORKDIR}
```

Reply:
I will make the change in the next PR.